SUPPORT_POLICY.md

```diff
@@ -0,0 +1,27 @@
+Distroless is designed to follow debian on their [standard support timeline](https://wiki.debian.org/DebianReleases#Production_Releases).
```

Comment:
Remove ` designed `? The use of Debian is an implementation detail.  The Debian release information is surfaced in the tags, but more to prevent breaches from Hyrum’s Law. Diatribes could switch to Nix (with a lot of implementation work) and most users wouldn’t know the difference.

Reply:
> Diatribes

Phenomenal typo.

Reply:
Also Nix while probably more convenient doesn't yet have full range of architecture support.

Reply:
The Debian package metadata is exposed in the images and is consumed by some users for dependency/vulnerability scanning.

Reply:
right, this will also be published in an sbom of some sort -- in the future.